components/collections/src/codepointinvlist/cpinvlist.rs

```diff
@@ -253,13 +260,15 @@ impl<'data> CodePointInversionList<'data> {
     ///     assert_eq!(&invalid, &actual);
     /// }
     /// ```
-    pub fn try_from_inversion_list_slice(inv_list: &'data [u32]) -> Result<Self, InvalidSetError> {
-        let inv_list_zv: ZeroVec<u32> = ZeroVec::from_slice_or_alloc(inv_list);
+    pub fn try_from_inversion_list_slice(
```

Comment:
Yeah, probably delete this. People should make a ZeroVec and use `try_from_inversion_list`. It was more interesting with u32s for a number of reasons.